cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -0,0 +1,193 @@
+* Only a single AWS account that will be used for service billing can be associated with a Red Hat account. Typically an organizational AWS account that has other AWS accounts, such as developer accounts, linked would be the one that is to be billed, rather than individual AWS end user accounts.
+* Red Hat accounts belonging to the same Red Hat organization will be linked with the same AWS account. Therefore, you can manage who has access to creating {hcp-title} clusters on the Red Hat organization account level.
+
+. Complete the Red Hat account linking after reviewing the terms and conditions:
++
+[NOTE]
+====
+This step is available only if the logged-in Red Hat account, or the Red Hat organization managing the Red Hat account, was not linked to an AWS account before.
+====
++
+image::rosa-rh-account-connection-8.png[]
++
+Both the Red Hat and AWS account numbers are shown on this screen.
+
+. Click the *Connect accounts* button if you agree with the service terms.
++
+If this is the first time you are using the {hybrid-console}, you will be asked to agree with the general managed services terms and conditions before being able to create the first ROSA cluster:
++
+image::rosa-terms-conditions-9.png[]
++
+Additional terms that need to be reviewed and accepted will be shown after clicking the *View Terms and Conditions* button:
++
+image::rosa-terms-conditions-9-5.png[]
++
+Submit your agreement once you have reviewed any additional terms when prompted at this time.
+
+. The {hybrid-console-second} provides a confirmation that AWS prerequisites were completed and lists the first steps needed for cluster deployment:
++
+image::rosa-cluster-create-10.png[]
+
+. The following steps pertain to technical deployment of the cluster:
++
+image::rosa-deploy-11.png[]
++
+* It is possible that these steps will be performed on a different machine than where the service enablement and account linking were completed.
+* As mentioned previously, any Red Hat account belonging to the Red Hat organization that was linked with the AWS account that activated the ROSA service will have access to creating a cluster and will be able to select the billing AWS account that was linked under this Red Hat organization previously.
+
+The last section of this page shows cluster deployment options, either using the `rosa` CLI or though the web interface:
++
+image::rosa-cli-ui-12.png[]
++
+* The following steps describe cluster deployment using the `rosa` CLI.
+* If you are interested in deployment using the web interface only, you can skip to the _{hcp-title} cluster deployment using the web interface_ section. However, note that the `rosa` CLI is required for certain tasks, such as creating the account roles. If you are deploying ROSA for the first time, follow this the CLI steps until running the `rosa whoami` command, before skipping to the web interface deployment steps.
+
+== {hcp-title} cluster deployment using the CLI
```

Comment:
Reader Commentary: Feels weird that this guide for activating your account would then go into yet another start up guide for clusters.

Could we just link to the other docs for this?